### DISCUSSION TITLE
Can I add a Lightroom preset to a button?

### DISCUSSION THREAD

**Discussion by hendrikm2** [rsjaffe/MIDI2LR#744] [General]
Hi,

I´tried the search function, but no avail.

I use the german version of LR and Midi2LR (5.4.0.0), but I should be fine with the english names.

I want to attach on of the adaptive Lightroom presets ("Adaptiv: Porträt verbessern" probably "Adaptive: Improve Portrait"). I looked in the settings of Midi2LR but didn´t find it. 
Is ist possible? Where is it hidden?

Thanks, Hendrik (and sorry for starting every sentence with the letter "I")

![LRscreen](https://github.com/rsjaffe/MIDI2LR/assets/69892245/0340e42c-588b-4d03-9c27-be558b0f4ea3)

**Comment by MarcksHouse** [comment]
hello,
If you add the preset to your favorites you may see it appear in the list no ?

![image](https://github.com/rsjaffe/MIDI2LR/assets/144238913/99437ad3-081d-4859-b628-26e4732a4332)
